setup.py

```diff
@@ -34,7 +34,7 @@
 AUTHOR = "Henrik Blidh"
 
 # What packages are required for this module to be executed?
-REQUIRED = ["requests", "six"]
+REQUIRED = open("requirements.txt").read().splitlines()
```

Comment:
I do not like reading the requirements.txt from setup.py, but until I convert to setup.cfg and pyproject.toml, I can accept this.

Reply:
I understand, having them duplicated was worse in my opinion. Agree on pyproject.toml is the way to fix it properly.